.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015"]
+}
```

Comment:
Can you please remove this file? [`babel-it`](https://github.com/IonicaBizau/babel-it) takes care of babelifying.